sdks/java/core/src/main/java/org/apache/beam/sdk/coders/AvroCoder.java

```diff
@@ -334,11 +315,8 @@ public void verifyDeterministic() throws NonDeterministicException {
   /**
    * Returns a new {@link DatumReader} that can be used to read from an Avro file directly. Assumes
    * the schema used to read is the same as the schema that was used when writing.
-   *
-   * @deprecated For {@code AvroCoder} internal use only.
    */
   // TODO: once we can remove this deprecated function, inline in constructor.
-  @Deprecated
```

Comment:
Can we make this method package private and marked with @VisibleForTesting now since they were marked @Deprecated for that reason